lnrpc/rpc.proto

```diff
@@ -1716,13 +1713,6 @@ message Invoice {
     */
     string payment_request = 9 [json_name = "payment_request"];
 
-    /**
```

Comment:
We currently don't use the description hash field, but unlike the receipt field, it's actually part of BOLT 11. It's tagged field `h`. Atm, is specified, it _will_ be stored in the database in the form of the payreq itself. As a result, removing it would break existing behavior that works, unlike removing the receipt which doesn't do anything at all atm.

Reply:
Suppose we _would_ use the description hash field, what would it be used for in `lnd`? Is there more to it than store and return?

Reply:
To commit to anything that's greater than the 600 byte or so limit of the normal description field. Or it can be used to store a 256-bit identifier for an invoice. In these cases, it wouldn't be used _directly_ by `lnd`, but then again, many of the invoice fields like the description are like that: `lnd` stores it so users can query against it and implement higher level logic.

Reply:
Yes, that is what I mean. Just trying to properly interpret https://github.com/lightningnetwork/lnd/pull/2263#discussion_r240857344

But it seems that description hash should not be touched at all, correct?

Reply:
Yep, when I made that comment I forgot it was a part of the invoice itself.